test/new_test.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import unittest
+import json
+import os
+import platform
+from os.path import join as path_join
+
+
+from galore.cross_sections import get_csv_file_path
+from galore.cross_sections import _cross_sections_from_csv_data
+
+##obtain directory of current test file
+test_dir = os.path.abspath(os.path.dirname(__file__))
+
+##obtain expected directory of data-storing file of different systems
+if platform.system() == "Windows":
+    correct_directory = os.path.join(os.getenv('LOCALAPPDATA'), 'galore_data')
+else:
+    correct_directory = os.path.join(os.path.expanduser('~'), '.galore_data')
+
+
+class test_get_cross_sections_from_data_table(unittest.TestCase):
+    '''To check function _cross_sections_from_csv_data works well with csv datatable'''
+
+    def test_get_cross_sections_from_scofield_datatable(self):
+    ##Import datatable stored in json file and convert them to the correct form, which mocks csv datatable for element Al
+        json_path = path_join(test_dir, 'Al_test_scofield.json')
+        with open(json_path) as json_file:
+            data = json.load(json_file)
+        data['data_table'] = np.array(data['data_table'])
+        data['electron_counts'] = np.array(data['electron_counts'])
+
+    ##check the function goes well with above datatable
+        cross_sections_scofield,_= _cross_sections_from_csv_data(800,data,'Scofield')
+        self.assertAlmostEqual(cross_sections_scofield['s'],4.41385e-06)
+    
+
+    def test_get_cross_sections_from_yeh_datatable(self):
+    
+        ##Import datatable stored in json file and convert them to the correct form, which mocks csv datatable for element Al
+        json_path = path_join(test_dir, 'Al_test_yeh.json')
+        with open(json_path) as json_file:
+            data = json.load(json_file)
+        data['data_table'] = np.array(data['data_table'])
+        data['electron_counts'] = np.array(data['electron_counts'])
+
+    ##check the function goes well with above datatable
+        cross_sections_scofield,_= _cross_sections_from_csv_data(800,data,'Yeh')
+        self.assertAlmostEqual(cross_sections_scofield['s'], 0.00145)
```

Comment:
Should this variable be `cross_sections_yeh`? These tests are nearly duplicated, which suggests they might be more cleanly implemented by calling a common function.

(This is not urgent, I can clean it up later if need be.)